sentry-core/src/main/java/io/sentry/core/transport/AsyncConnection.java

```diff
@@ -359,8 +368,8 @@ public void run() {
               .getLogger()
               .log(
```

Comment:
M: More convinced now that it would be nice to have some static log methods we could import and use here, that take `@Nullable Object hint` and log these, we use it a lot

Reply:
yes, all of them are gonna be public though, they are being used in different packages.